src/components/00-materials/scripts/export-svg.js

```diff
@@ -14,80 +14,84 @@ const camelCase = _camelCase =>
   camelise(componentName(_camelCase.replace('.', '-')).replace(/-/g, ' '));
 const toClassName = _className =>
   camelCase(_className).replace(/^\w/, c => c.toUpperCase());
+const cleanFileName = _fileName => {
+  return _fileName
+    .replace(/ /g, '-')
+    .replace(/&/, '')
+    .replace(/\(/, '')
+    .replace(/\)/, '')
+    .replace(/^([\d-])/, '_$1') // add _ if first character is a digit
+    .toLowerCase();
+};
 
 const importIconsPath = './icons-raw';
 const exportIconsPath = './icons';
 
 const importImagesPath = './images-raw';
 const exportImagesPath = './images';
 
-fs.readdir(importIconsPath, (err, items) => {
-  if (err) {
-    throw new Error('Cannot find icons');
-  }
-
-  let namedExports = '';
-  items.forEach(icon => {
-    if (/\.svg/.test(icon)) {
-      const iconPath = path.resolve(importIconsPath, icon);
-      const contents = fs
-        .readFileSync(iconPath, 'utf8')
-        .replace(/(\r\n|\n|\r)/gm, '');
-      const fileName = path.basename(icon);
-      const className = toClassName(fileName);
+function readdirAndSaveSvgToJs(importPath, exportPath) {
+  fs.readdir(importPath, (err, items) => {
+    if (err) {
+      throw new Error(`Cannot find files. Error: ${err}`);
+    }
 
-      if (!fs.existsSync(exportIconsPath)) {
-        fs.mkdirSync(exportIconsPath);
+    let namedExports = '';
+    items.forEach(icon => {
+      if (/\.svg/.test(icon)) {
+        const iconPath = path.resolve(importPath, icon);
+        const contents = fs
+          .readFileSync(iconPath, 'utf8')
+          .replace(/(\r\n|\n|\r)/gm, '');
+
+        let fileName = path.basename(icon);
+        fileName = cleanFileName(fileName);
+        const className = toClassName(fileName);
+
+        // Generate a js file forEach svg file found
+        fs.writeFileSync(
```

Comment:
It doesn't make sesne to use synced methods here. But I leave it up to you to correct that, sicne I'm not sure how often we use that.

Reply:
:D